mysql-test/suite/galera/t/galera_password_history.cnf

```diff
@@ -0,0 +1,4 @@
+!include ../galera_2nodes.cnf
+
+[mysqld]
+password_history=10
```

Comment:
Not a strong opinion though! Can we move this part to user creation? This can reduce one file from our repo.
```
CREATE USER IF NOT EXISTS 'test1'@'%' IDENTIFIED WITH 'mysql_native_password' AS '*6C387FC3893DBA1E3BA155E74754DA6682D04747' PASSWORD HISTORY 10;
```

Reply:
done